docs/integrations/sources/googleanalytics.md

```diff
@@ -1,44 +1,37 @@
 # Google Analytics
 
-## Overview
```

Comment:
we can remove this whole page because v4 is the only version supported going forward

Reply:
Cool, done - removed from SUMMARY.md first and will purge later.